Library/Formula/eet.rb

```diff
@@ -34,4 +36,18 @@ def install
                           "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    cp "#{pkgshare}/examples/eet-basic.c", testpath
+    eina = Formula["eina"]
+    flags = %W[
+      -I#{include}/eet-1
+      -I#{eina.include}/eina-1
+      -I#{eina.include}/eina-1/eina
+      -L#{lib}
+      -leet
+    ]
```

Comment:
You can give the flags directly to `system` below; we don’t need an intermediate array.